packages/docusaurus-search-local/src/server/index.ts

```diff
@@ -358,6 +365,12 @@ export const tokenize = (input) => lunr.tokenizer(input)
             // Do not index error page.
             return [];
           }
+          if (ignore?.includes(url)) {
+            // Do not index pages in 'ignore' config.
+            actuallyIgnored.add(url);
+            initialIgnore.delete(url);
```

Comment:
Shouldn't we use \`/${route}\` instead of `url` here, or am I missing something?